tests/unit/test_calibration.py

```diff
@@ -143,7 +178,7 @@ def test_calibration_successful() -> None:
     offset = np.array([0.1, 0.2, 0.1])
     expected_scale = 1 / scale
     expected_offset = -offset / scale
-    dummy_no_motion = np.random.randn(1000, 3) - 0.5
+    dummy_no_motion = np.random.randn(1000, 3)
```

Comment:
Why is this value changed?

Reply:
Accidentally saved a version when I was playing around with values for the tests when initially adding z-score method + gradient descent. Switched it back now. (Same for above)